packages/adminrouter/extra/src/README.md

```diff
@@ -117,13 +136,13 @@ includes
         └── master.conf
 ```
 
-All NGINX related configuration directives reside in the `includes` directory which
-contains directories reflecting all the sections present in NGINX configuration
-(http|main|server). Because they are always present, no
-matter the flavour/server type, they are chosen as top-level directories
-in the `includes` dir. The `common.conf`, `agent.conf` and `master.conf`
-files are flavour-agnostic thus they reside in each section's main dir
-and are present/included only in the Open AR repository. EE repository re-uses them
+All NGINX related configuration directives reside in the `includes` directory
+which contains directories reflecting all the sections present in NGINX
+configuration (http|main|server). Because they are always present, no matter
+the flavour/server type, they are chosen as top-level directories in the
+`includes` dir. The `common.conf`, `agent.conf` and `master.conf` files are
```

Comment:
`$s/dir/directory`.

In the general case, let's not optimise for typing fewer characters in individual words.